mvlearn/semi_supervised/ctregression.py

```diff
@@ -0,0 +1,580 @@
+# python version 3.7.6
```

Comment:
Add the license at the top of this file (just copy paste from the ctclassifier.py file for example)